src/main/groovy/nebula/plugin/publishing/verification/StatusSchemaAttributeRule.groovy

```diff
@@ -1,25 +1,23 @@
 package nebula.plugin.publishing.verification
 
+import groovy.transform.CompileStatic
 import org.gradle.api.artifacts.CacheableRule
 import org.gradle.api.artifacts.ComponentMetadataContext
 import org.gradle.api.artifacts.ComponentMetadataDetails
 import org.gradle.api.artifacts.ComponentMetadataRule
 
 
 //@CacheableRule TODO: this is disable to test RealisedMavenModuleResolveMetadataSerializationHelper duplicate objects
+@CompileStatic
```

Comment:
This helps so we don't need to do the check? Interesting :-)

Reply:
So more context is here: https://github.com/gradle/gradle/issues/9847#issuecomment-508445431